README.md

```diff
@@ -84,3 +84,5 @@ system.
 We are still structuring an organized contribution process, but we more than
 welcome proposed changes through Pull-Requests. For cataloged issues, you can
 check our [issues page](https://github.com/kworkflow/patch-hub/issues).
+Don't forget to read our
+[Contribuitor's Guide](https://kworkflow.org/content/howtocontribute.html)
```

Comment:
Nice idea! However, we don't strictly follow the conventions of `kw`, because (1) most things don't translate one being in Bash and the other in Rust, and (2) some "rules" are different, like using the semantic commit messages with `fix/refactor/feat/docs/style/...`. With that being said, we truly need to update this "How to Contribute" section...